Sources/SKTestSupport/UnknownLanguage.swift

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+
+public extension Language {
+  static let unknown: Language = Language(rawValue: "unknown")
```

Comment:
Is this used anywhere other than the line
```
checkCoding(Language.unknown, json: "\"unknown\"")
```

in the CodingTests.swift above the one you removed?  If so, we don't need to expose this static property at all and can just check coding on `Language(rawValue: "unknown")`.

Reply:
done